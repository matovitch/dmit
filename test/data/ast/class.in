class A
{
    a : i64;
    b : f32;
}
