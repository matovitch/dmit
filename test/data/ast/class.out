{"node":"Module","path":{},"parentPath":{},"imports":[],"definitions":[{"node":"Definition","status":"LOCAL","value":{"node":"Class","name":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"A"}},"members":[{"node":"Type Claim","variable":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"a"}},"type":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"i64"}}},{"node":"Type Claim","variable":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"b"}},"type":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"f32"}}}]}}],"modules":[]}